setup.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pathlib
+import setuptools
+
+# The directory containing this file
+CWD = pathlib.Path(__file__).parent
+
+# The text of the README file
+README = (CWD / "README.md").read_text()
+
+# Get the list of dependencies from the requirements.txt file
+with open(os.path.join(CWD, "requirements.txt")) as requirements_file:
+    # Parse requirements.txt, ignoring any commented-out lines.
+    REQUIREMENTS = [
+        line
+        for line in requirements_file.read().splitlines()
+        if not line.startswith("#")
+    ]
+
+version = "0.1.0"
+assert "." in version
+
+setuptools.setup(
+    name="dap-py",
+    version=version,
+    description="Packages for Jupyter Notebook users to interact with data from A2e, Livewire, and the SPP data platform.",
+    long_description=README,
+    long_description_content_type="text/markdown",
+    url="https://github.com/DAP-platform/dap-py",
+    author="Sherman Beus, Logan Dihel, Michael Kieburtz, Matthew Macduff, Devin McAllester, Eliot Quon, Paulo Yamagata",
+    author_email="michael.kieburtz@pnnl.gov",
+    license="MIT",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Natural Language :: English",
+        "Programming Language :: Python :: 3.5",
```

Comment:
I noticed some of the code uses f-strings, which were only introduced in 3.6, thus 3.5 will not work. I would also recommend dropping support for python 3.6 because that reached its end of life from Python almost a year ago.